libmscore/scorefile.cpp

```diff
@@ -1488,8 +1488,10 @@ void Score::writeSegments(Xml& xml, int strack, int etrack,
                         cr->writeTuplet(xml);
                         }
                   if ((segment->segmentType() == Segment::Type::EndBarLine) && (m->mmRestCount() < 0 || m->mmRest())) {
+                        bool mmRestOn = styleB(StyleIdx::createMultiMeasureRests);
                         BarLine* bl = static_cast<BarLine*>(e);
-                        bl->setBarLineType(m->endBarLineType());
+                        if(mmRestOn)
```

Comment:
spece between if and (. I'd not introduce a new variable for just a single use, so'd rather do it like 
`if (styleB(StyleIdx::createMultiMeasureRests))
      bl->setBarLineType(m->endBarLineType()); `